projects/js-packages/publicize-components/src/components/connection-management/snackbars.tsx

```diff
@@ -0,0 +1,28 @@
+import { SnackbarList } from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { store as noticesStore } from '@wordpress/notices';
+import styles from './style.module.scss';
+
+// Last three notices. Slices from the tail end of the list.
+const MAX_VISIBLE_NOTICES = -3;
```

Comment:
Nitpick, but maybe for readability of the variable name, we could have this as `3` and negate it in the slicing itself?

Reply:
Yeah, sure. I actually copied this component as is from gutenberg.